### PR TITLE
Improve warning about missing pnctl

### PR DESCRIPTION
**Pull request by CarlSchwan**

### CODE REVIEW

**Review by GretaD — APPROVED** [review]

File: console.php [hunk 1]
```diff
@@ -86,6 +86,7 @@ function exceptionHandler($exception) {
 
 	if (!function_exists('pcntl_signal') && !in_array('--no-warnings', $argv)) {
 		echo "The process control (PCNTL) extensions are required in case you want to interrupt long running commands - see https://www.php.net/manual/en/book.pcntl.php" . PHP_EOL;
+		echo "Additionally the function 'pcntl_signal' and 'pcntl_signal_dispatch' need to be enabeld in your php.ini." . PHP_EOL;
```
Comment (come-nc) [inline console.php:89]: Maybe add a `function_exists` tests for `pcntl_signal_dispatch` as well then?

**Review by come-nc — APPROVED** [review]

File: console.php [hunk 2]
```diff
@@ -84,8 +84,9 @@ function exceptionHandler($exception) {
 		exit(1);
 	}
 
-	if (!function_exists('pcntl_signal') && !in_array('--no-warnings', $argv)) {
+	if ((!function_exists('pcntl_signal') || function_exists('pcntl_signal_dispatch')) && !in_array('--no-warnings', $argv)) {
```
Comment (CarlSchwan) [inline console.php:87]:
```suggestion
	if ((!function_exists('pcntl_signal') || !function_exists('pcntl_signal_dispatch')) && !in_array('--no-warnings', $argv)) {
```

**Review by kesselb — APPROVED** [review]

**Pull request merged**